be/src/exec/pipeline/pipeline_driver.cpp

```diff
@@ -78,7 +78,13 @@ Status PipelineDriver::prepare(RuntimeState* runtime_state) {
     _local_rf_holders = fragment_ctx()->runtime_filter_hub()->gather_holders(all_local_rf_set);
 
     for (auto& op : _operators) {
-        RETURN_IF_ERROR(op->prepare(runtime_state));
+        int64_t time_spent = 0;
+        {
+            SCOPED_RAW_TIMER(&time_spent);
+            RETURN_IF_ERROR(op->prepare(runtime_state));
+        }
+        op->set_prepare_time(time_spent);
```

Comment:
Can the time cost of operator::prepare be ignored?

Reply:
Sometimes, some operators' preparation time costs 10~50ms, such as scan and aggregate. 
This timer could help us to find this.